SoftLayer/managers/hardware.py

```diff
@@ -50,16 +50,16 @@ def __init__(self, client, ordering_manager=None):
         else:
             self.ordering_manager = ordering_manager
 
-    def cancel_hardware(self, hardware_id, reason='unneeded', comment='',
+    def cancel_hardware(self, network_id, reason='unneeded', comment='',
```

Comment:
You seam to have renamed `hardware_id` to `network_id` in the entire hardware manager file... That doesn't seem to make sense. Find/replace error?